packages/atlas/src/providers/uploadsManager/store.ts

```diff
@@ -26,11 +32,13 @@ type UploadStoreActions = {
   setUploadStatus: (contentId: string, status: Partial<UploadStatus>) => void
   addAssetFile: (assetFile: AssetFile) => void
   setIsSyncing: (isSyncing: boolean) => void
-  removeProcessingAssetId: (contentId: string) => void
-  addProcessingAssetId: (contentId: string) => void
+  removeProcessingAsset: (contentId: string) => void
+  addProcessingAsset: (contentId: string) => void
   addNewChannelId: (channelId: string) => void
 }
 
+const THREE_MINUTES = 1000 * 60 * 3
```

Comment:
Let's call this something like `UPLOAD_PROCESSING_TIMEOUT` and move to global config. I would also change this to one minute I think